### PR TITLE
Add and style PR badge for PPP

### PR DESCRIPTION
**Pull request by RichDom2185**
Before and after:

image

Syntax is similar to autoglossary:
```markdown
<!-- instead of: -->
* Update website styling ([PR#333](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/333))
<!-- we have: -->
* Update website styling [[PR#333]]
```

Also has hover effects:
image

### CODE REVIEW

**Review by Eugenetanwl3881 — DISMISSED** [review]
LGTM. Thanks!

**Review by Eugenetanwl3881 — APPROVED** [review]

File: docs/team/eugenetanwl3881.md [hunk 1]
```diff
@@ -16,48 +16,46 @@ Given below are my contributions to the project.
 
   * What it does: allows the user to tag a valid item with a valid tag (PR [\#171](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/171), [\#172](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/172))
```
Comment (bryanljx) [inline docs/team/eugenetanwl3881.md:17]: Are PR links like these supposed to be changed to the new format as well?

**Review by bryanljx — APPROVED** [review]
Looks good, only that some PR links seem to be not change to the new format.

File: docs/team/eugenetanwl3881.md (hunk 1 above)
Comment (RichDom2185) [reply docs/team/eugenetanwl3881.md:17]: I'll leave it to the individual members to ensure it's formatted correctly. I just did a simple ctrl+f to fix all those that are already in the correct format as specified by the PPP checker (PR#XXX). Any other format, I'll leave it to them.

**Pull request merged**